services/QuillLMS/app/models/teacher_notification_setting.rb

```diff
@@ -34,7 +34,7 @@ def self.notification_types
   validates :notification_type,
     presence: true,
     inclusion: { in: ->(model) { model.class.notification_types },
-      message: '%<value>s is not a valid TeacherNotification type' }
```

Comment:
I find that extra whitespace on `message` can be removed by moving `in:` down a line:

```
validates :notification_type,
  presence: true,
  inclusion: { 
    in: ->(model) { model.class.notification_types },
    message: '%<value>s is not a valid TeacherNotification type' 
  }
```

It also makes it more clear that `in:` and `message:` are both args on inclusion.